### DISCUSSION TITLE
How to force blank line between import statements and class declaration?

### DISCUSSION THREAD

**Discussion by knopp10000** [scalameta/scalafmt#3926] [Q&A] [answered]
Is it possible to force a blank line after the last import statement in all files?

So that code that looks like this:

```
package org.scalafmt

import scala.collection.Map
case class x(y) = {
```

Looks like:

```
package org.scalafmt

import scala.collection.Map

case class x(y) = {
```

Thanks in advance.

**Comment by tgodzik** [comment, marked as answer]
I think:
```
newlines.topLevelStatementBlankLines = [
  {
    blanks { before = 1 }
  }
]
``` 
should help.

You might want to take a look at https://scalameta.org/scalafmt/docs/configuration.html#newlinestoplevelstatementblanklines
